xm-commons-permission/src/main/java/com/icthh/xm/commons/permission/repository/PermittedRepository.java

```diff
@@ -73,10 +73,12 @@
         String selectSql = format(SELECT_ALL_SQL, entityClass.getSimpleName());
         String countSql = format(COUNT_ALL_SQL, entityClass.getSimpleName());
 
-        String permittedCondition = createPermissionCondition(privilegeKey);
-        if (StringUtils.isNotBlank(permittedCondition)) {
-            selectSql += WHERE_SQL + permittedCondition;
-            countSql += WHERE_SQL + permittedCondition;
+        Collection<String> permittedCondition = createPermissionCondition(privilegeKey);
+
+        if (isNotEmpty(permittedCondition)) {
+            String orChainedPermittedCondition = String.join(" OR ", permittedCondition);
```

Comment:
wrap each part to () for keep semantic
a mean that after join we must have (<part1>) OR (<part2>) instead of <part1> OR <part2>

Reply:
added